tools/matchms/matchms_fingerprint_similarity.xml

```diff
@@ -0,0 +1,84 @@
+<tool id="matchms_fingerprint_similarity" name="matchms fingerprint similarity" version="@TOOL_VERSION@+galaxy0" profile="21.09">
+    <description>matchms fingerprint similarity calculation</description>
```

Comment:
Can you provide a more explanatory description?

Reply:
I improved the description in https://github.com/RECETOX/galaxytools/pull/321/commits/97a56eb90529ee48fbae42a8dd338d47faca51df and also added an additional footnote with a link to the details about the used molecular fingerprint